WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -498,6 +499,20 @@ public void onEventMainThread(CoreEvents.MainViewPagerScrolled event) {
     }
 
     @SuppressWarnings("unused")
+    public void onEventMainThread(UploadService.UploadErrorEvent event) {
+        SiteModel site = getSelectedSite();
+        if (site != null) {
+            if (event.post.getLocalSiteId() == site.getId()) {
```

Comment:
Do we need to null check `event.post` here?

Reply:
uh, nice! addressed in 0ad299b